### PR TITLE
fix(agw): Prerequisites in Readme.

### PR DESCRIPTION
**Pull request by Alt-Shivam**
## Summary
Added Prerequisites section in Agw-Helm charts README, to make it easy for user to start deployment of AGW.

Signed-off-by: Shivank Chaudhary shivank1128@gmail.com

### CODE REVIEW

**Comment by Alt-Shivam** [comment]
I can add a reference to this repo [GoTo](https://github.com/ShubhamTatvamasi/magma-galaxy) for automated install of required components [GoTo](https://github.com/ShubhamTatvamasi/magma-galaxy/blob/master/deploy-orc8r.yml#L5:L9).
If it is feasible.

File: lte/gateway/deploy/agwc-helm-charts/README.md [hunk 1]
```diff
@@ -17,6 +17,13 @@ The following table list the configurable parameters of the agw chart and their
 | `config.domainName` | Orchestrator domain name. | `` |
 | `persistant.name` | Secret name containing agwc certs. | `agwc-claim` |
 
+## Pre-requisites
+
+- A Kubernetes cluster with the following components deployed:
+    - MetalLB
+    - OpenEBS
```
Comment (ShubhamTatvamasi) [inline lte/gateway/deploy/agwc-helm-charts/README.md:24]: These are only needed on a private cluster, not in public.

File: lte/gateway/deploy/agwc-helm-charts/README.md [hunk 2]
```diff
@@ -17,6 +17,13 @@ The following table list the configurable parameters of the agw chart and their
 | `config.domainName` | Orchestrator domain name. | `` |
 | `persistant.name` | Secret name containing agwc certs. | `agwc-claim` |
 
+## Pre-requisites
```
Comment (mpfirrmann) [inline lte/gateway/deploy/agwc-helm-charts/README.md]: Typo: Its "Prerequisites" without the dash

**Review by mpfirrmann — CHANGES_REQUESTED** [review]

File: lte/gateway/deploy/agwc-helm-charts/README.md (hunk 2 above)
Comment (Alt-Shivam) [reply lte/gateway/deploy/agwc-helm-charts/README.md]: Fixed

**Review by mpfirrmann — APPROVED** [review]

**Comment by ajahl** [comment]
Please follow the [PR Guidlines](https://github.com/magma/magma/wiki/Contributing-Code#commit-and-pull-request-guidelines) and change PR title so that the `Check for semantic PR title` CI check can pass.

**Review by ajahl — APPROVED** [review]

**Review by ShubhamTatvamasi — APPROVED** [review]

**Comment by voisey** [comment]
Hi @Alt-Shivam, so we can merge this, we need to get the DCO check green. It's currently red (see https://github.com/magma/magma/actions/runs/3090415722/jobs/5186265721) due to the commits not being signed off. Could you please address this? :slightly_smiling_face: Also see https://github.com/magma/magma/wiki/Contributing-Code#signed-off-commits for further information.

**Comment by Alt-Shivam** [comment]
Added this change in #14182 
Thanks

**Pull request closed without merging**